Mathlib/RingTheory/DedekindDomain/AdicValuation.lean

```diff
@@ -55,14 +55,13 @@ We are only interested in Dedekind domains with Krull dimension 1.
 dedekind domain, dedekind ring, adic valuation
 -/
 
-
 noncomputable section
 
 open scoped Multiplicative
 
 open Multiplicative IsDedekindDomain
 
-variable {R : Type*} [CommRing R] [IsDedekindDomain R] {K : Type*} [Field K]
+variable {R : Type*} [CommRing R] [IsDedekindDomain R] {K L : Type*} [Field K] [CommSemiring L]
```

Comment:
Can you call the ring `S`? Just for psychological reasons (`L` is surely a field!).